.nvmrc

```diff
@@ -0,0 +1 @@
+v10.15.3
```

Comment:
What about using `lts/*` -> defaults to the latest LTS version

Source: https://github.com/creationix/nvm#nvmrc

Reply:
Nice 👍 will update